examples/js/loaders/draco/DRACOLoader.js

```diff
@@ -14,24 +14,35 @@
 //
 'use strict';
 
-THREE.DRACOLoader = function(manager) {
+// |dracoPath| sets the path for the Draco decoder source files. The default
+// path is "./". If |dracoDecoderType|.type is set to "js", then DRACOLoader
+// will load the Draco JavaScript decoder.
+THREE.DRACOLoader = function(dracoPath, dracoDecoderType, manager) {
+    this.timeLoaded = 0;
     this.manager = (manager !== undefined) ? manager :
         THREE.DefaultLoadingManager;
     this.materials = null;
     this.verbosity = 0;
-    this.dracoDecoderType = {};
+    this.attributeOptions = {};
+    this.dracoDecoderType =
+        (dracoDecoderType !== undefined) ? dracoDecoderType : {};
+    this.drawMode = THREE.TrianglesDrawMode;
+    this.dracoSrcPath = (dracoPath !== undefined) ? dracoPath : './';
+    THREE.DRACOLoader.loadDracoDecoder(this);
```

Comment:
How about adding an option to set the decoder explicitly:

```js
var loader = new THREE.DRACOLoader(decoderType, manager);
loader.setDecoder(decoder);
loader.load(function () {
  // ...
});
```

And then either (a) making the constructor `decoderPath` optional, or (b) only providing it through a similar `setDecoderPath(path)` method.

This would (I think) provide an opt-out for users with a bundler, who don't necessarily control their folder structure.

Thoughts @mrdoob?

Reply:
In the latest commit, I added the condition that if `DracoDecoderModule` is defined then skip loading the decoder file. I think it could somehow solve this issue. But still looking at how to make it work with wasm.